doc/release/2.8.1.rst

```diff
@@ -0,0 +1,283 @@
+Tarantool 2.8.1
+===============
+
+Release: :tarantool-release:`2.8.1`
+
+Date: 2021-04-21 Tag: 2.8.1
+
+Overview
+--------
+
+2.8.1 is the :doc:`beta </release/policy>`
+version of the 2.8 release series.
+
+This release introduces 28 new features and resolves 31 bugs since the
+2.7.2 version. There can be bugs in less common areas. If you find any,
+feel free to `report an
+issue <https://github.com/tarantool/tarantool/issues>`__ on GitHub.
+
+Notable changes are:
+
+-  Implement ability to run multiple iproto threads.
+-  Set box.cfg options with environment variables.
+-  Introduce box.ctl.promote() and the concept of manual elections.
+-  Lua memory profiler enhancements.
+
+Compatibility
+-------------
+
+Tarantool 2.x is backward compatible with Tarantool 1.10.x in the binary
+data layout, client-server protocol, and replication protocol.
+
+Please
+`upgrade <https://www.tarantool.io/en/doc/latest/book/admin/upgrades/>`__
+using the ``box.schema.upgrade()`` procedure to unlock all the new
+features of the 2.x series.
+
+Functionality added or changed
+------------------------------
+
+Core
+~~~~
+
+-   Introduce exclude_null option to index part definition. When it is
+    on, index filters and doesn’t store tuples with “null” value of this
+    part (:tarantool-issue:`4480`).
```

Comment:
`` :tarantool-issue:`xxxx` `` is rendered as `tarantool/tarantool#xxxx` and there are a lot of such links in release notes. Can we shrink it to just `gh-xxxx` in release notes? We can introduce another clause for the short form.

Reply:
Yes, we can. The template is set in the config https://github.com/tarantool/doc/blob/7fd17c61a920f84f06bd76dbb5dc3dc604633819/conf.py#L40. But what about links to issues in other repos?

Reply:
That's why I said about introducing another template rather than modifying this one: it is obvious in this context that the gh-xxxx link means an issue in tarantool. In other contexts we can use the full form (usually the link density is not so tight and the full tarantool/tarantool#xxxx form fits perfectly).

Just like autolinks on GitHub: the link to the same repository is renreder to the short form, while links to another ones are rendered with a full repo name:

```
* https://github.com/tarantool/doc/issues/1
* https://github.com/tarantool/tarantool/issues/1
```

->

* https://github.com/tarantool/doc/issues/1
* https://github.com/tarantool/tarantool/issues/1